nomenclature/codelist.py

```diff
@@ -134,6 +134,14 @@ def cast_variable_components_args(cls, v, values):
 
         return v
 
+    @validator("mapping")
+    def check_stray_tag(cls, v, values):
```

Comment:
Thanks for creating the pydantic validator :+1:. You don't need the `values` input as you're only looking at one value in isolation.
```suggestion
    def check_stray_tag(cls, v):
```